src/app/components/core/configuration-manager/configuration-group/configuration-group.component.ts

```diff
@@ -239,21 +239,25 @@ export class ConfigurationGroupComponent implements AfterViewInit {
         e.group === 'Basic' ? acc.unshift(e) : acc.push(e);
         return acc;
       }, []);
+
+    // set advance as a first tab if no default config
```

Comment:
does not make any sense! Use case?

Reply:
when there is only readonly property in the default configuration, in that case no group for default configuration and on modal window dynamic property there first tab need to be set on first advance group

Reply:
I see what  you mean, Though it will be the rarest to see that; good to safeguard that